src/DefaultFileSystem.cpp

```diff
@@ -75,11 +76,35 @@ DefaultFileSystem::Read(const std::string& path) const
   return result;
 }
 
+std::string tmpfile(std::string pathnormalized)
```

Comment:
Actually I guess we can get rid of this method and put it in `DefaultFileSystem::Write` if we don't need a second version for wide string. Nevertheless, by the way, we start method names with a capital letter.